src/lib/coda_metrics/coda_metrics.ml

```diff
@@ -213,6 +213,41 @@ module Network = struct
   (* TODO: track non gossip RPC messages as well *)
 end
 
+module Snark_work = struct
+  let subsystem = "Snark_work"
+
+  let snark_work_received_gossip : Counter.t =
+    let help = "# of snark work received from peers" in
```

Comment:
should be "# of snark bundles" or something. "snark work" is easy to confuse with the jobs.

Reply:
I was using `work` for completed work and `job` for work that needs to be completed. So maybe snark work bundles and snark job bundles? :)

Reply:
how about completed snark works? People who are not Deepthi are not going to immediately understand these terms